crates/lib/src/compiler/isa/qubit.rs

```diff
@@ -170,7 +170,14 @@ impl FrbSim1q {
 const DEFAULT_DURATION_RX: f64 = 50.0;
 
 fn rx_gates(node_id: i64, frb_sim_1q: &FrbSim1q) -> Result<Vec<Operator>, Error> {
-    let fidelity = frb_sim_1q.fidelity_for_qubit(node_id)?;
+    let fidelity = match frb_sim_1q.fidelity_for_qubit(node_id) {
```

Comment:
style: `.or_else` may be more succinct

Reply:
Unfortunately, `or_else` doesn't let us distinguish between error types, so it's not possible to say "fallback to 0.0 for `Error::MissingQubitBenchmark` but not for any other kind of error".